sw/device/silicon_creator/lib/boot_data.h

```diff
@@ -68,11 +69,24 @@ typedef struct boot_data {
    * The BL0 slot that is prioritized during boot.
    */
   uint32_t primary_bl0_slot;
+  /**
+   * Next owner key (ECDSA).
```

Comment:
nit: Add that this key is don't care or cleared when `ownership_state` is `UNLOCKED_ANY`?

Reply:
Fixed.